hack/redis_install.sh

```diff
@@ -88,10 +88,30 @@ if [ `uname -s` == "Linux" ]; then
     sudo sed -i -e "s/^appendonly no$/appendonly yes/g" $REDIS_CONF_Ubuntu
     sudo egrep -v "(^#|^$)" $REDIS_CONF_Ubuntu |egrep "(appendonly |appendfsync )"
 
+    # === Enable Redis server remote support
+    sudo sed -i -e "s/^bind 127.0.0.1 -::1$/bind 0.0.0.0/g" $REDIS_CONF_Ubuntu
+    sudo egrep -v "(^#|^$)" $REDIS_CONF_Ubuntu |egrep "(bind 0.0.0.0)"
+
+    sudo sed -i -e "s/^protected-mode yes$/protected-mode no/g" $REDIS_CONF_Ubuntu
+    sudo egrep -v "(^#|^$)" $REDIS_CONF_Ubuntu |egrep "(protected-mode no)"
+    # === Enable Redis server remote support
+
+    # Restart redis-server as a service
     sudo ls -al /lib/systemd/system/ |grep redis
 
     sudo systemctl restart redis-server.service
     sudo systemctl status redis-server.service
+
+    # Check whether network port 6379 is listened
+    sudo netstat -nlpt | grep 6379
+
+    # Add Ubuntu Firewall rule to open Redis port 6379
+    sudo ufw enable << EOF
+y
+EOF
+    sudo ufw status
+    sudo ufw allow 6379/tcp
```

Comment:
port 6379 needs to be opened at the GCP level as well to allow remote connections.

Reply:
ufw is Ubuntu firewall level regardless AWS or GCP

Reply:
Remove ufw openning.